raiden-dapp/src/components/notification-panel/NotificationArea.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <v-snackbar
+    v-model="notification.display"
+    :timeout="notification.duration"
+    top
+    centered
+    vertical
+    app
+    rounded
+    max-width="550px"
+    @input="notificationShown(notification.id)"
+  >
+    <v-row no-gutters class="notification-area__notification__content">
```

Comment:
This object has a lot in common with `NotificationCard`, maybe the implementation could be shared? Dismiss/clear differences could be done through events and props

Reply:
Also, no need to hard push it this PR if too complex, but `NotificationPayload` could have an optional `route` or `url` which, when set, could make this whole element a link to the respective route. e.g. Receive Disabled notification above could direct to the UDC screen.

Reply:
This would also help bring the icons to the notifications panel

Reply:
I am not sure of the total time available to finish it but maybe it makes more sense to do that in a new PR